.eslintrc

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
I used recommended pack here and turned off some rules (made "warn" for them instead of error), also fixed code for some rules as well